src/shared/components/palette/case-link/write-case-link-field.component.ts

```diff
@@ -48,4 +48,8 @@ export class WriteCaseLinkFieldComponent extends AbstractFieldWriteComponent imp
     }
     return new RegExp('^\\b\\d{4}[ -]?\\d{4}[ -]?\\d{4}[ -]?\\d{4}\\b$').test(valueString.trim());
   }
+
+  public inputBlur() {
+    this.caseReferenceControl.markAsUntouched();
```

Comment:
Are you sure this is what you want to do? If you enter an invalid value and then leave the field, the errors won't show, based on the logic within the HTML file (`*ngIf="caseReferenceControl.errors && caseReferenceControl.touched"`).

This bit of code ensures that `caseReferenceControl.touched` will always evaluate to `false` when the control doesn't have focus.

Reply:
As discussed changed it to on dirty state